Codemine/Extensions/CGRect+Utilities.swift

```diff
@@ -35,7 +35,7 @@ public extension CGRect {
      
      - returns: a new CGRect with the width and height reversed to those of the current one
      */
-    public func rectByReversingSize() -> CGRect {
+	public var reversingSize: CGRect {
```

Comment:
Oh, I missed that. I guess it's fine then, if that's what Apple does. Also, why is that BezierPath thing a func and not a var?